resource-monitoring/README.md

```diff
@@ -0,0 +1,40 @@
+## OBJECTIVE
+How about monitoring all you system resources through real-time graphs?.Do you know this could be done just by a small effort - lets start monitoring! :open_mouth:
+
```

Comment:
@shivam99aa the target audience doesn't know about application they know about CPU and RAM there system is consuming right now. Does this looks better ? 

How about monitoring all your system resources through real-time graphs?.Do you know  running too many applications on your system it may start running slow. You want to know what is running on your system - lets start monitoring! :open_mouth: